CMSgov/mint-app
# [TASK-EASI-3695] - FE model participation questions

# EASI-3695 - FEATURE
# EASI-3743 - TASK

## Changes and Description

- Updated queries to use new schema fields - `participationInModelPrecondition` fields
- Updated page 1 of General Characteristics to use new fields
- Updated readonly to use new fields
- Updated snaps
- Updated unit test
- Updated cypress tests




## How to test this change

- Navigate to page 1 of General Characteristics
- Fill out all the variations of the `participationInModelPrecondition` fields 
  -  Ensure they match figma
- Verify Readonly data
- Verify csv data

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
